src/core/queryClient.ts

```diff
@@ -335,8 +402,15 @@ export class QueryClient {
     queryFn: QueryFunction<TQueryFnData, TQueryKey>,
     options?: FetchInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryKey>
   ): Promise<InfiniteData<TData>>
-  fetchInfiniteQuery<TQueryFnData, TError, TData = TQueryFnData, TQueryKey extends QueryKey = QueryKey>(
-    arg1: TQueryKey | FetchInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
+  fetchInfiniteQuery<
+    TQueryFnData,
+    TError,
+    TData = TQueryFnData,
+    TQueryKey extends QueryKey = QueryKey
+  >(
+    arg1:
+      | TQueryKey
+      | FetchInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
```

Comment:
not sure why prettier re-formatted all of this all of a sudden. looks like we are not checking prettier formatting as part of the pipeline 🤔 .

I will make a second PR that adds prettier formatting to the pipeline to avoid unnecessary diffs.